roles/ora-host/tasks/main.yml

```diff
@@ -433,3 +431,28 @@
     - "{{ listdisks }}"
   when: asm_disk_management == "asmlib"
   tags: asm-disks
+  
+  - name: (asmlib) ASM listing results
+  debug:
+    msg:
+    - "{{ item.stderr_lines }}"
+    - "{{ item.stdout_lines }}"
+  with_items:
+    - "{{ asmconfig }}"
+    - "{{ listdisks }}"
+  when: asm_disk_management == "asmlib"
+  tags: asm-disks
+
+- name: Block http to metadata 
```

Comment:
This looks like a separate change;  I think it applies only to RAC instances;  can this be reflected in the rule?  As well, iptables is very much legacy, so the recommended way of applying this kind of rule would be to use firewalld;  perhaps something like

    - name: Block metadata service
# Ansible's firewalld only allows output reules via the "direct" interface, and ansible doesn't yet support it
      shell: |
        firewall-cmd --direct --add-rule ipv4 filter OUTPUT 0 -d 169.254.169.254 -j REJECT --reject-with icmp-host-unreachable
        firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 0 -d 169.254.169.254 -j REJECT --reject-with icmp-host-unreachable